### PR TITLE
Add new settings to meeting model

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

File: docs/example-data.json [hunk 1]
```diff
@@ -194,6 +194,8 @@
 
         "conference_show": false,
         "conference_auto_connect": false,
+        "conference_los_restriction": false,
+        "stream_url": "",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:198]: prefix `conference_`?

File: docs/models.txt [hunk 2]
```diff
@@ -139,22 +139,27 @@ Interface committee {
...
     description: string;
     location: string;
     start_time: datetime;
-    end_time: datetime;
-    welcome_title: string;
-    welcome_text: HTML;
+    end_time: datetime; // TODO: is this needed?
     custom_translations: JSON;
 
+    // System
```
Comment (FinnStutzenstein) [inline docs/models.txt:153]: I'm currently not sure, if those settings should be editable in the settings page, or managed in some kind of committee-detail/meeting-list -> THis is mainly a UI decision, but for now please put them into the settings

File: docs/example-data.json (hunk 1 above)
Comment (jsangmeister) [reply docs/example-data.json:198]: Strictly, this isn't a setting for the conference, that's why in OS3 the prefix isn't present either. But I can add it if you want

File: docs/models.txt (hunk 2 above)
Comment (jsangmeister) [reply docs/models.txt:153]: Will do

File: docs/example-data.json (hunk 1 above)
Comment (jsangmeister) [reply docs/example-data.json]: Done

**Pull request merged**